plugins/inputs/uwsgi/uwsgi_test.go

```diff
@@ -0,0 +1,125 @@
+package uwsgi_test
```

Comment:
please also write a test checking that errors are returned from Gather properly. You can just use invalid addresses for http, tcp, unix to test that.